UnrealPlugin/ThirdPersonMPGSDKSetup.md

```diff
@@ -246,6 +255,27 @@ bool UMyGameInstance::OnGSDKHealthCheck()
     UE_LOG(LogTemp, Warning, TEXT("Healthy!"));
     return true;
 }
+
+void UMyGameInstance::OnGSDKServerActive()
+{
+    /**
+     * Server is transitioning to an active state.
+     * Optional: Add in the implementation any code that is needed for the game server when
+     * this transition occurs.
+     */
+    UE_LOG(LogTemp, Warning, TEXT("Active!"));
+}
+
+void UMyGameInstance::OnGSDKReadyForPlayers()
+{
+    /**
+     * Server is transitioning to a StandBy state. Game initialization is complete and the game is ready
+     * to accept players.
+     * Optional: Add in the implementation any code that is needed for the game server before
+     * initialization completes.
+     */
+    UE_LOG(LogTemp, Warning, TEXT("Finished Initialization - Moving to StandBy!"));
```

Comment:
This should definitely be in a separate PR, but...

We should not be using LogTemp in code we're giving to customers. We should create a log channel, PlayFabGsdkLog or something like that, and log to our own channel.
Please file a separate bug, and fix in a separate PR.
(Unless an appropriate log channel already exists, in which case, we need to stop using LogTemp and switch all our logs to our own channel)

Reply:
It exists already I believe. Good catch, I will file it as separate PR.